Sources/MlemMiddleware/Content Models/Instance/Instance3Providing.swift

```diff
@@ -39,7 +39,7 @@ public extension Instance3Providing {
     var administrators_: [Person2]? { instance3.administrators }
     
     
-    func addAdmin(person: any Person, added: Bool) async throws {
-        instance3.administrators = try await api.addAdmin(person, added: added)
+    func addAdmin(personId: Int, added: Bool) async throws {
+        try await api.addAdmin(personId: personId, added: added)
     }
```

Comment:
In similar cases I've overloaded methods like this to also accept `Person` for convenience

```suggestion
    func addAdmin(personId: Int, added: Bool) async throws {
        try await api.addAdmin(personId: personId, added: added)
    }
    
    func addAdmin(_ person: any Person, added: Bool) async throws {
        addAdmin(personId: person.id, added: added)
    }
```